os/arch/arm/src/armv7-a/arm_dataabort.c

```diff
@@ -183,6 +183,8 @@ uint32_t *arm_dataabort(uint32_t *regs, uint32_t dfar, uint32_t dfsr)
 
 segfault:
 	if (!IS_SECURE_STATE()) {
+		/* Add new line to distinguish between normal log and assert log.*/
+		lldbg_noarg("\n");
```

Comment:
As Chang mentioned, pls move this to the start of the print function to avoid repitition. 
Also, if you are adding comment for this, then you can change it as follows
`Abort log must always start at a new line`